src/worldstate/package.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>worldstate</name>
+  <version>0.0.0</version>
+  <description>The world-state package</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag -->
```

Comment:
In general, we try and cut down `package.xml` files so they're short and readable (a fully commented version is always available in `sample_package`). For an example of this, see [here](https://github.com/UBC-Snowbots/Snowflake/blob/master/src/sb_pointcloud_processing/package.xml)